src/Commands/MachineToken/DeleteCommand.php

```diff
@@ -22,6 +22,36 @@ class DeleteCommand extends TerminusCommand
      *   Removes the given machine token from the user's account
      */
     public function delete($machine_token_id) {
+        $user = $this->session()->getUser();
+
+        // Find the token. Will throw an exception if it doesn't exist.
+        $machine_token = $user->machine_tokens->get($machine_token_id);
+        if (empty($machine_token)) {
+            throw new \Exception(vsprintf('There are no machine tokens with the id %s.', [$machine_token_id]), 1);
```

Comment:
@TeslaDethray: Is this the right way to handle command failures? Do we want to use more specific Exceptions (`TerminusNotFoundException` for example).

I'm not in love with gluing the id into the error message before adding it to the Exception for language translation and management reasons but this is fine for now I guess.